style.css

```diff
@@ -4,14 +4,15 @@ html * {
 
```

Comment:
[ OPTIONAL ] Please use a directory for all of the images in the project directory. This way project directory looks clean and managed.